client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -611,6 +630,19 @@ export default {
       dplan.notify.notify('confirm', Translator.trans('recommendation.pasted'))
     },
 
+    addClickListenerToImages () {
+      const images = this.$refs.recommendationContainer.querySelectorAll('img')
+      images.forEach(img => {
+        img.addEventListener('click', this.imageClicked)
+      })
+    },
+
+    imageClicked ({ target }) {
+      this.clickedImg.src = target.src
+      this.clickedImg.alt = target.alt
+      this.$refs.imgModal.toggle()
+    },
+
```

Comment:
since this code is now duplicated, I suggest to move the logic in a separate Component `ImageModal`.

Reply:
I moved it to new component ImageModal. Should this be in demosplan-ui?

Reply:
Done here [6d377f9](https://github.com/demos-europe/demosplan-core/pull/3394/commits/6d377f9b480fa70421436893c99cbee563d99937) and here [e7cc439](https://github.com/demos-europe/demosplan-core/pull/3394/commits/e7cc4395b90405f21b1e38533fb165bbc345ac9f)

Reply:
> Should this be in demosplan-ui?

Good Question. I think Yes it could, but we should leave it here for the moment and refactor it in a second step

Reply:
> I moved it to new component ImageModal. 

Thanks. Much better imho